### PR TITLE
Multi blob key fix

### PR DESCRIPTION
**Pull request by ttt-77**
1. fix parsing a WHERE clause to filter rows based on combinations of values in multiple columns
2. fix using full scan engine bug when all samples are needed in approximate aggregation

### CODE REVIEW

**Comment by ttt-77** [comment]
@ddkang Could you please review this PR?

File: aidb/engine/base_engine.py [hunk 1]
```diff
@@ -294,21 +294,24 @@ def add_filter_key_into_query(
     param query_expression: sqlglot parsed query expression
     '''
     selected_column = []
-    col_name_set = set()
+    col_name_list = []
```
Comment (ddkang) [inline aidb/engine/base_engine.py:297]: Why was this changed to a list?

File: tests/test_aggregation_gaussian_mk.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
+from multiprocessing import Process
```
Comment (ddkang) [inline tests/test_aggregation_gaussian_mk.py:1]: Can you explain what this test does in a comment in this file?

**Review by ddkang — COMMENTED** [review]
Overall looks okay, can you also add a test for a non-AQP with multiple IN clauses?

File: aidb/engine/base_engine.py (hunk 1 above)
Comment (ttt-77) [reply aidb/engine/base_engine.py:297]: Because choosing certain row in pd.Dataframes requires a list format. And in line 304, we have a condition to avoid duplicate, so the logic doesn't change.
https://github.com/ddkang/aidb/blob/f556532746d97f85d170172a974261af6d00cc2e/aidb/engine/base_engine.py#L304

**Comment by ttt-77** [comment]
> Overall looks okay, can you also add a test for a non-AQP with multiple IN clauses?

Emm, I think previous code doesn't support IN operator in WHERE clause as described in #45.  It only supports such 'where(b.timestamp > 10 or c.color = red) and (b.timestamp < 12 or c.color = red)' clause. 
I can fix it in another PR if you want.

**Comment by ddkang** [comment]
Got it. Can you raise an issue or add context to the other issue?

**Review by ddkang — APPROVED** [review]
